org.lflang/src/org/lflang/generator/ReactorInstance.xtend

```diff
@@ -147,17 +147,17 @@ class ReactorInstance extends NamedInstance<Instantiation> {
      * If the port reference has the form `c.x`, where `c` is a bank of reactors,
      * then the list will contain the port instances belonging to each bank member.
      * 
-     * If the cross argument is false, then for any port reference `b.m` where `b`
-     * is a bank and `m` is a multiport, this function iterates over bank members first,
+     * If a given port reference `b.m`. where `b` is a bank and `m` is a multiport,
+     * is unqualified, this function iterates over bank members first,
      * then ports. E.g., if `b` and `m` have width 2, it returns
      * `[b0.m0, b0.m1, b1.m0, b1.m1]`. 
      * 
-     * If the cross argument is true, then for any port reference `b.m` where `b`
-     * is a bank and `m` is a multiport, this function iterates over ports first,
+     * If a given port reference `b.m`. where `b` is a bank and `m` is a multiport,
+     * is qualified with 'interleaved', this function iterates over ports first,
```

Comment:
Suggest:
If a given port reference has the form `interleaved(b.m)`, where `b` is a bank and `m` is a multiport, then this function iterates over ports first, ...